.github/workflows/ci.yaml

```diff
@@ -17,6 +17,8 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version: ${{ matrix.go-version }}
+      - name: Install STACKIT CLI
+        uses: jkroepke/setup-stackit-cli@v1
```

Comment:
We may want to fork this to keep dependencies clean.

But I like to get other opinions on that.

Reply:
I could use a simple curl here as well.